qiskit/transpiler/passes/routing/algorithms/bip_model.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, dag, coupling_map, qubit_subset, dummy_timesteps=None):
                 "BIPMappingModel assumes the same size of virtual and physical qubits."
             )
 
-        self._index_to_virtual = dict(enumerate(dag.qubits))
+        self._index_to_virtual = {q: dag.find_bit(q).index for q in dag.qubits}
```

Comment:
Actually, I missed this one. Here we're not creating a `qubit: index` map, but an `index: qubit` map. Later indices are iterated through, so I don't think there's any performance advantage to using `find_bit` on this one. I'd just leave the original, so there would be no changes in this file.